src/js/global-nav.js

```diff
@@ -274,7 +276,30 @@ function addListeners(breakpoint, wrapper) {
   overlay.addEventListener('click', closeNav);
 }
 
-export const createNav = ({ maxWidth = '68rem' } = {}) => {
+function validURL(str) {
+  const pattern = new RegExp(
+    '^(https?:\\/\\/)?' + // protocol
```

Comment:
I think this has to start with http(s):// (or else append it) otherise you can have links like `hiring: "example.com"` which would result in "http://dev.local:8300/example.com".

Reply:
Removed the validation as there might be a few ways we want to link like interal anchors so safer without it.